.vscode/extensions.json

```
@@ -0,0 +1,9 @@
+{
+	"recommendations": [
+		"streetsidesoftware.code-spell-checker",
+		"streetsidesoftware.code-spell-checker-portuguese-brazilian",
+		"editorconfig.editorconfig",
+		"dbaeumer.vscode-eslint",
```

Playwright VS code extension is a very useful tool to debug and write tests

```suggestion
		"dbaeumer.vscode-eslint",
		"ms-playwright.playwright",
```

---

Think playwright is more situational. For example, BE devs won't use it :trollface:

---

Agreed with Kevin. 

But also doesn't hurt. Playwright is used a lot in the project so i think makes sense to include. Anyone not wanting can ignore.